packages/storage/Readme.md

```diff
@@ -1,5 +1,286 @@
 # Secret Contract Development Toolkit - Storage Tools
 
-⚠️ This package is a sub-package of the `secret-toolkit` package. Please see its crate page for more context.
+⚠️ This package is a sub-package of the `secret-toolkit` package. Please see its crate page for more context. You need Rust 1.61+ to compile this package.
 
-This package contains all the tools related to storage access patterns
+This package contains many tools related to storage access patterns. This readme file assumes basic familiarity with basic cosmwasm storage, [click here to learn about this](https://docs.scrt.network/secret-network-documentation/development/secret-contracts/storage).
+
+## **How to Import This Subpackage**
+
+To import this package, add one of the following lines to your `Cargo.toml` file
+
+```toml
+secret-toolkit = { version = "0.3", default-features = false, features = ["utils"] }
+secret-toolkit-storage = "0.3"
+secret-toolkit-serialization = "0.3"
+```
+
+for the release versions, or
+
+```toml
+secret-toolkit = { git = "https://github.com/scrtlabs/secret-toolkit.git", default-features = false, features = ["utils"] }
+secret-toolkit-storage = { git = "https://github.com/scrtlabs/secret-toolkit.git" }
+secret-toolkit-serialization = { git = "https://github.com/scrtlabs/secret-toolkit.git" }
```

Comment:
usually when pointing to git you want to specify a branch name, tag, or commit hash